apps/openmw/mwphysics/physicssystem.cpp

```diff
@@ -1152,9 +1152,9 @@ namespace MWPhysics
 
                 bool waterCollision = false;
                 if (effects.get(ESM::MagicEffect::WaterWalking).getMagnitude()
-                        && cell->getCell()->hasWater()
-                        && !world->isUnderwater(iter->first.getCell(),
-                                               osg::Vec3f(iter->first.getRefData().getPosition().asVec3())))
+                        && cell->getCell()->hasWater())
+//                        && !world->isUnderwater(iter->first.getCell(),
+//                                               osg::Vec3f(iter->first.getRefData().getPosition().asVec3())))
```

Comment:
Don't forget too that it's very hard to arrive into a new project, things
that you would be able to do in a few minutes because you know the code
take a few days to new comers... It just helps when you show instead of
just saying, and after that people try to do their best next time.
Precious is probably not the right word to talk about my time, but I still
see it as a waste of time anyway !
Anyway I made a new branch, walk_on_water, because with this you can have a
very very tiny patch just removing a few lines to make all this to work
flawlessly.

2015-10-02 9:24 GMT+02:00 Bret Curtis <notifications@github.com>:

> In apps/openmw/mwphysics/physicssystem.cpp
> <https://github.com/OpenMW/openmw/pull/761#discussion_r40996604>:
>
> > @@ -1152,9 +1152,9 @@ namespace MWPhysics
> >
> >                  bool waterCollision = false;
> >                  if (effects.get(ESM::MagicEffect::WaterWalking).getMagnitude()
> > -                        && cell->getCell()->hasWater()
> > -                        && !world->isUnderwater(iter->first.getCell(),
> > -                                               osg::Vec3f(iter->first.getRefData().getPosition().asVec3())))
> > +                        && cell->getCell()->hasWater())
> > +//                        && !world->isUnderwater(iter->first.getCell(),
> > +//                                               osg::Vec3f(iter->first.getRefData().getPosition().asVec3())))
>
> We appreciate it that you want to help, but with a project of this size,
> it makes our job easier when everyone follows the same coding guidelines.
>
> I bring this up because you ignored scrawl and mine's advice twice now, 17
> days ago and again yesterday for now comments.
>
> We would rather bounce the patch back to your with critique to make it
> better and to ease you into our workflow.
>
> As for wasting your time, I'm sorry about that. We had no idea that your
> time was worth more than our time.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenMW/openmw/pull/761/files#r40996604>.
>